controllers/postController.js

```diff
@@ -100,4 +102,159 @@ const list = async (req, res) => {
   }
 };
 
-module.exports = { upsert, list };
+const today = async (req, res) => {
+  if (!isValidString(req.params.keywordId) || isEmptyString(req.params.keywordId)) {
+    return res.status(400).send({ message: "[InvalidKeywordId] Error occured" });
+  }
+
+  const hasKeywordId = (await keywordModel.findOne({ _id: req.params.keywordId }).exec()) !== null;
+  if (!hasKeywordId) {
+    return res.status(400).send({ message: "[NotExistedKeywordId] Error occured" });
+  }
+
+  const keywordId = req.params.keywordId;
+
+  try {
+    const today = new Date();
+    const yesterday = new Date(today.getFullYear(), today.getMonth(), today.getDate() - 1);
+
+    const todayPostCount = await postModel
+      .find({ keywordId })
+      .find({
+        createdAt: {
+          $gte: today.setHours(0, 0, 0, 0),
+          $lt: today.setHours(23, 59, 59, 999),
+        },
+      })
+      .countDocuments()
+      .exec();
+    const yesterdayPostCount = await postModel
+      .find({ keywordId })
+      .find({
+        createdAt: {
+          $gte: yesterday.setHours(0, 0, 0, 0),
+          $lt: yesterday.setHours(23, 59, 59, 999),
+        },
+      })
+      .countDocuments()
+      .exec();
+
+    return res.status(200).json({
+      todayPostCount,
+      diffPostCount: todayPostCount - yesterdayPostCount,
+    });
+  } catch {
+    return res
+      .status(500)
+      .send({ message: "[ServerError] Error occured in 'postController.today'" });
+  }
+};
+
+const postCount = async (req, res) => {
+  if (!isValidString(req.params.keywordId) || isEmptyString(req.params.keywordId)) {
+    return res.status(400).send({ message: "[InvalidKeywordId] Error occured" });
+  }
+
+  const keywordInfo = await keywordModel.findOne({ _id: req.params.keywordId }).exec();
+  if (keywordInfo === null) {
+    return res.status(400).send({ message: "[NotExistedKeywordId] Error occured" });
+  }
```

Comment:
```suggestion
  const isExistedKeyword = (await keywordModel.findOne({ _id: req.params.keywordId }).exec()) !== null;
  if (!isExistedKeyword) {
    return res.status(400).send({ message: "[NotExistedKeywordId] Error occured" });
  }
```
특정 데이터 존재여부를 판단하는 코드 컨벤션을 정해보면 어떨까요? :) 

1. if 조건문의 조건에 변수 혹은 함수의 반환값을 두어 가독성 높이기
   - 현재 `findOnde`으로 반환된 `keywordInfo`를 사용하고 있는 경우가 가장 하단 response 외에 없어서 `keywordInfo` 대신 변수명으로 반환되어도 괜찮다고 생각했어요. (=> 이렇게 수정되면 함께 수정할 수 있는 부분을 요 리뷰 맨아래 기재했어요)
2. 변수명을 부정 의미를 가급적 담지 않도록 하기 (`isNotExistedKeyword` 👎 / `isExistedKeyword` 👍)
   - 가령 `!isNotExistedKeyword`를 활용할 때 부정의 부정은 가독성에 좋지 않기 떄문입니다.

여러 컨트롤러에서 재사용 되다보니 함께 고려해보아요! 이번 광고 필터링 PR에도 포함되어 있어요. cc. @SsongQ-92 
https://github.com/Team-Bloblow/Bloblow-Server/blob/86bbe4b0e2e988155143616f139d8939c4cb8114/controllers/keywordController.js#L18-L21

추가
- 관련해서 163번 라인도 구조분해할당을 미리 해둘 수 있구요!
```js
const { keywordId, keyword } = req.params.keywordId;
```

Reply:
저도 처음엔 `hasKeywordId` 로 명명하여 해당 `keywordId`의 존재유무를 파악하여 `null`인지 아닌지 확인했었는데요.
```jsx
const hasKeywordId = (await keywordModel.findOne({ _id: req.params.keywordId }).exec()) !== null;
```
저희가 URL을 통해 `keywordId`만 받고 있기 때문에, `keyword 이름`을 response로 전달해주기 위해선 
하단에서 다시 `find`쿼리를 사용하기 보단 상단의 존재여부 로직을 변경하여 null 체크 -> 키워드 정보 확인 로직으로 변경했던 거였습니다. 
제가 의도한 바는 이러한데, 이거에 대해선 어떻게 생각하실까요? 
@jin-ttao @SsongQ-92

Reply:
설명해주셔서 감사해요. 구조분해할당으로 가져오지 못하는 부분을 제가 체크 못했네요 :) 현재 코드를 유지해도 좋겠습니다 👍